### PR TITLE
Kiruththigan

### PR DESCRIPTION
**Pull request by kiruththigan**

### CODE REVIEW

File: text.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (ramz7) [inline text.txt]: what is this text?

File: defect-tracker-server/src/main/java/com/defect/tracker/controller/EmployeeController.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+					HttpStatus.BAD_REQUEST);
+		}
+
+		if (file.isEmpty()) {
+			return new ResponseEntity<>(new ValidationFailureResponse(ValidationConstance.EMPLOYEE_PROFILE_EMPTY,
+					validationFailureStatusCode.getEmpProfileIsEmpty()), HttpStatus.BAD_REQUEST);
+		}
+
+		if (!file.getContentType().equals("image/jpeg")) {
+			System.out.println(validationFailureStatusCode.getEmpProfileContenetTypeException());
```
Comment (ramz7) [inline defect-tracker-server/src/main/java/com/defect/tracker/controller/EmployeeController.java:75]: remove this print

File: defect-tracker-server/src/main/java/com/defect/tracker/controller/EmployeeController.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+							validationFailureStatusCode.getEmpProfileSizeException()),
+					HttpStatus.BAD_REQUEST);
+		}
+
+		if (file.isEmpty()) {
+			return new ResponseEntity<>(new ValidationFailureResponse(ValidationConstance.EMPLOYEE_PROFILE_EMPTY,
+					validationFailureStatusCode.getEmpProfileIsEmpty()), HttpStatus.BAD_REQUEST);
+		}
+
+		if (!file.getContentType().equals("image/jpeg")) {
```
Comment (ramz7) [inline defect-tracker-server/src/main/java/com/defect/tracker/controller/EmployeeController.java:74]: this validation for?

File: defect-tracker-server/src/main/java/com/defect/tracker/data/entities/Employee.java [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Entity
+@Table(name = "employee")
+public class Employee {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+	private String name;
+	private String designation;
```
Comment (ramz7) [inline defect-tracker-server/src/main/java/com/defect/tracker/data/entities/Employee.java]: there is a table designation. you can it after merge designation to development.so then use designation-id

File: text.txt (hunk 1 above)
Comment (kiruththigan) [reply text.txt]: this remove next time

File: defect-tracker-server/src/main/java/com/defect/tracker/controller/EmployeeController.java (hunk 3 above)
Comment (kiruththigan) [reply defect-tracker-server/src/main/java/com/defect/tracker/controller/EmployeeController.java:74]: for image file

File: defect-tracker-server/src/main/java/com/defect/tracker/util/ValidationConstance.java [hunk 5]
```diff
@@ -2,7 +2,21 @@
 
 public class ValidationConstance {
 
-	public static final String PROJECT_EXISTS = "Project  Already Exists";
+	public static final String PROJECT_EXISTS = "project  already Exists";
+
+	public static final String EMPLOYEE_EMAIL_EXISTS = "employee email  already Exists";
+
+	public static final String EMPLOYEE_Id_EXISTS = "employee id  already Exists";
```
Comment (ramz7) [inline defect-tracker-server/src/main/java/com/defect/tracker/util/ValidationConstance.java]: please change  first letter of  all words to upper case

File: defect-tracker-server/src/main/java/com/defect/tracker/data/entities/Employee.java (hunk 4 above)
Comment (kiruththigan) [reply defect-tracker-server/src/main/java/com/defect/tracker/data/entities/Employee.java]: ok

File: defect-tracker-server/src/main/java/com/defect/tracker/util/ValidationConstance.java (hunk 5 above)
Comment (kiruththigan) [reply defect-tracker-server/src/main/java/com/defect/tracker/util/ValidationConstance.java]: ok

File: defect-tracker-server/src/main/java/com/defect/tracker/controller/EmployeeController.java (hunk 2 above)
Comment (kiruththigan) [reply defect-tracker-server/src/main/java/com/defect/tracker/controller/EmployeeController.java:75]: image validation  no need remove this validation

**Review by rishaahamadh — APPROVED** [review]

**Pull request merged**